### ISSUE TITLE
Documents should be revised.

### ISSUE THREAD

**Issue by NJUJYB** [coin-or/Ipopt#359]
I have installed Ipopt based on Windows 7 guided by [Installation with Cygwin using GNU compilers](https://www.coin-or.org/Ipopt/documentation/node15.html#CygwinInstall) and [Ipopt-Installation](https://coin-or.github.io/Ipopt/INSTALL.html). However, there are some drawbacks, which would be misleading and should be fixed in the documents.
1. The codes for compiling third-party HSL has already existed in ThirdParty/HSL after conducting ./configure based on Ipopt-3.12.13. Therefore, there is no need to download coin-or-tools. Furthermore, downloaded third-party codes should be organized as ThirdParty/HSL/coinhsl instead of ThirdParty-HSL/coinhsl.
2. There is no 'Select required packages (RECOMMENDED)' in the progress of Cygwin installation. Although Cygwin will still install related packages according to the dependency. Packages like make, cmake, autoconf should be selected by user.
3. There is no gcc4 listed in 'Devel', although it is listed in the website of Cygwin. Therefore, gcc-core, gcc-fortran should be selected instead.
4. During the configuration progress of ipopt by using ./configure, g++ and BLAS are both checked and required. Therefore, gcc-g++, liblapack-dev should also be selected during the installation of Cygwin.
5. The directory C:\cygwin should be also revised according to the version of Cygwin (e.g., Cygwin64).

**Comment by svigerske** [comment]
The instructions under the first link you mention are outdated. I tried to remove them by deleting them from the corresponding repository, but that didn't have the desired effect - I haven't heard back from the admin of the coin-or server regarding this problem.

The instructions under the second link (https://coin-or.github.io/Ipopt/INSTALL.html) are the current one, but are for Ipopt 3.13.0. Things have a changed a lot compared to 3.12.13, including differences on how to build HSL (your point 1).
The instructions on how to build under Cygwin have been removed in the new docu as hadn't been updated for a long while and I don't intend to do so. Msys2 is the preferred tool to build Ipopt executables or libraries for Windows.

**Comment by NJUJYB** [comment]
Thanks a lot. I have deployed AMPL based on Ipopt successfully for both Windows and Linux by using the executables directly from AMPL. I have also tried coibrew script before. But the options to other releases, e.g., release 3.12.13, misleaded me a lot. I think a freshman would easily lose his way when he faces multiple releases. I suggest to point out the differences and the recommended version of Ipopt within the ReadMe file directly.
